src/Stylist/Console/PublishAssetsCommand.php

```diff
@@ -56,15 +41,15 @@ public function handle()
      */
     protected function setupThemes()
     {
-        $this->app['events']->fire('stylist.publishing');
+        $this->laravel['events']->fire('stylist.publishing');
```

Comment:
There no reason to have `$this->app` because every command will receive `$this->laravel`.